### PR TITLE
Lakshminaraayana sf

### PR DESCRIPTION
**Pull request by lakshminaraayana**
👍

### CODE REVIEW

File: lakshminaraayana-sf.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+### SF
+
+#### Food 
+- Thai Idea
```
Comment (lakshminaraayana) [inline lakshminaraayana-sf.md:5]: sounds good

File: lakshminaraayana-sf.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+- Dosa
+
+#### Things to do :eyes:
+- Twin Peaks
```
Comment (lakshminaraayana) [inline lakshminaraayana-sf.md:10]: Like it

Comment (dyellow) [inline lakshminaraayana-sf.md:10]: Dosa, yum.

File: lakshminaraayana-sf.md (hunk 1 above)
Comment (lakshminaraayana) [inline lakshminaraayana-sf.md:5]: Thanks

**Pull request merged**